src/server/pps/persist/persist.proto

```diff
@@ -79,6 +86,16 @@ message UpdatePipelineStateRequest {
   string recent_error = 3;
 }
 
+message UpdatePipelineStoppedRequest {
+	string pipeline_name = 1;
+	bool stopped = 2;
```

Comment:
Nitpick - should use consistent whitespace. Looks like we use 2 spaces mostly in the file.